### DISCUSSION TITLE
Is it possible to install your own model or add with https://hugging face.co?

### DISCUSSION THREAD

**Discussion by superprogresor11** [Sanster/IOPaint#244] [General]
Is it possible to install your own model or add with https://hugging face.co?

**Comment by Sanster** [comment]
Yes! In next release, planing to add support for sd ckpt/safetensors models too. Stay tuned

**Reply by superprogresor11** [reply]
Will be waiting. There is a great lack of such an opportunity. It will be wonderful.

**Comment by Sanster** [comment]
Add load local inpainting model(ckpt/safetensor) support in 1.0.2, see full release note here: https://github.com/Sanster/lama-cleaner/releases/tag/1.0.0
